src/components/usMapContacts/style.ts

```diff
@@ -7,11 +7,20 @@ export const Container = styled.div`
   height: 500px;
   background: ${colors.white};
   box-shadow: 4px -4px 10px rgba(0, 0, 0, 0.5);
+
+  @media (max-width: 468px) {
+    width: 100%;
+    height: 270px;
+  }
 `;
 
 export const Components = styled.div`
   align-items: center;
   white-space: nowrap;
+
+  @media (max-width: 468px) {
+    display: none;
```

Comment:
It is brocken on screen 468px
![image](https://user-images.githubusercontent.com/6819965/207812290-2e501bfc-b0a9-43a8-8ed3-73a601b5b0f0.png)